configuration.rst

```diff
@@ -13,24 +20,19 @@ PostgREST reads a configuration file to determine information about the database
 
    Configuration can be reloaded without restarting the server. See :ref:`config_reloading`.
 
-The configuration file must contain a set of key value pairs. At minimum you must include these keys:
+The configuration file must contain a set of key value pairs. At a minimum you should include one of the following keys:
 
 .. code::
 
   # postgrest.conf
 
-  # The standard connection URI format, documented at
-  # https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
-  db-uri       = "postgres://user:pass@host:5432/dbname"
```

Comment:
This still seems like valuable information, specially for remote pg connections.

I think instead of replacing the old ways we should add a subsection like "default postgres connection".

Reply:
> This still seems like valuable information

Right. Looking at the whole changes, I think I removed a bit too much without replacement. Maybe the "database connection" section mentioned above could instead pick that up.

> I think instead of replacing the old ways we should add a subsection like "default postgres connection".

Hm. I would like to describe the default first and then go from there. That seems more natural to me.